services/conan/conan-version.tester.js

```diff
@@ -0,0 +1,8 @@
+import { isSemver } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
```

Comment:
Could you also add a not found case that has a bogus/non-existing target recipe just to track the validation of the error message?